externe_connecties/iv3_definitie_connectie.py

```diff
@@ -10,12 +10,18 @@ def geef_gebruikersvriendelijke_foutmeldingen(fouten):
     """
     Haal gebruikers vriendelijke vertalingen uit fouten.json voor fouten.
     """
```

Comment:
Ik denk dat je hier redelijke veel verschillende taken tegelijk uitvoert:

- Foutendefinitie ophalen
- Deserialiseren output van OSF controle
- Fouten voor mensen leesbaar maken

Misschien kleiner maken

Reply:
ja klopt, ik kan het wellicht simpeler maken door verschillende functies te maken

Reply:
ik heb iig de foutendefinitie ophalen er uit gehaald, heb ook meteen de file name aangepast, aangezien het nu geen iv_3_definitie connectie meer is. De deserialisatie vind ik niet per se nodig, het is het verschil tussen een dict en een list, hier komt dus een dict binnen als functie argument. Vind je dat een leesbaarheids-issue? of zie ik dit verkeerd en heb je een ander idee wat ik nog zou kunnen refactoren hier en waarom dat beter zou worden? hoor het graag